src/pages/tutorials/android-wallet.mdx

```diff
@@ -61,7 +61,7 @@ Once, you have added the JitPack repository, then in your app-level `build.gradl
 dependencies {
     // ...
     // focus-next-line
-    implementation 'com.github.web3auth:web3auth-android-sdk:7.4.0'
+    implementation 'com.github.web3auth:web3auth-android-sdk:10.0.0'
```

Comment:
Page needs revamp, the tutorial is outdated. Lot of breaking changes from 7.4.0, to 10.0.